x/deployment/client/cli/tx.go

```diff
@@ -32,10 +40,154 @@ func GetTxCmd(key string, cdc *codec.Codec) *cobra.Command {
 		cmdUpdate(key, cdc),
 		cmdClose(key, cdc),
 		cmdGroupClose(key, cdc),
+		makeItSoCommand(key, cdc),
 	)...)
 	return cmd
 }
 
+func makeItSoCommand(key string, cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "make-it-so [sdl-file]",
+		Short: fmt.Sprintf("Create %s", key),
```

Comment:
Still `Create deployment`, might want to add a little explanation here for the command.